docs/using-flatpak.rst

```diff
@@ -204,13 +237,50 @@ To fix inconsistencies with your local installation, use::
 
  $ flatpak repair
 
+This command is ran on the root installation by default, 
+append ``--user`` to repair the user installation.
+
 Flatpak also has a number of commands to manage the portal permissions of
 installed apps. To reset all portal permissions for an app, use ``flatpak
 permission-reset``::
 
- $ flatpak permission-reset org.gimp.GIMP
+ $ flatpak permission-reset your.application.ID
+
+There are more available flatpak permission commands, such as 
+``permission-remove``, ``permission-set``, and ``permission-show``.
+These can be managed with a GUI tool such as Flatseal(insert link to Flatseal here)
```

Comment:
Flatseal does not manage these permissions.

Reply:
Flatseal can show the app permissions, set them, and remove them. I believe this is exactly what it manages? Do I have that wrong?